interactive/notifications.py

```diff
@@ -31,6 +31,15 @@ def notify_analysis_request_submitted(analysis_request):
     slack.post(text=message)
 
 
+def notify_analysis_request_outputs_need_checking(issue_url):
```

Comment:
It's probably worth copying a lot of the text above in the slack message(in particular lines 26-29), as I expect the output checker may want to confirm the job ran successfully and may want to see the report in OSI as well. I also recall they like to know what codelist was used when checking the outputs too.

Reply:
That might be easiest, yes. I guess, the message going into interactive-requests is now just for information only, whereas previously someone needed to do something when it appeared. Whereas now, the only manual steps are to review the output and click on the send email button.

Reply:
@lucyb – updated to use the same message, emitted twice.  I've added a link to the created ticket to both as it was both simpler and it's probably useful to have all the information in one place?  Are you happy for this version to be merged?